debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorConfig.java

```diff
@@ -298,21 +301,64 @@
     public static final Field LOG_MINING_BUFFER_TYPE = Field.create("log.mining.buffer.type")
             .withDisplayName("Controls which buffer type implementation to be used")
             .withEnum(LogMiningBufferType.class, LogMiningBufferType.MEMORY)
+            .withValidation(OracleConnectorConfig::validateLogMiningBufferType)
             .withImportance(Importance.LOW)
             .withDescription("The buffer type controls how the connector manages buffering transaction data." + System.lineSeparator() +
                     System.lineSeparator() +
                     "memory - Uses the JVM process' heap to buffer all transaction data." + System.lineSeparator() +
                     System.lineSeparator() +
-                    "infinispan - This option uses an embedded Infinispan cache to buffer transaction data and persist it to disk." +
-                    " Use the log.mining.buffer.location property to define the location for storing cache files.");
+                    "infinispan_embedded - This option uses an embedded Infinispan cache to buffer transaction data and persist it to disk." + System.lineSeparator() +
```

Comment:
Is the old name still supported as a (deprecated) alias for "infinispan_embedded"?

Reply:
Yes, that was my intent just to make it easier for users on 1.7 to upgrade to 1.8.